lgc/state/TargetInfo.cpp

```diff
@@ -69,7 +69,7 @@ static void setGfx6BaseInfo(TargetInfo *targetInfo) {
   targetInfo->getGpuProperty().tessFactorBufferSizePerSe = 4096;
 
   // TODO: Accept gsOnChipDefaultLdsSizePerSubgroup from panel option
-  targetInfo->getGpuProperty().gsOnChipDefaultLdsSizePerSubgroup = 8192; // GFX6-8 value
+  targetInfo->getGpuProperty().gsOnChipDefaultLdsSizePerSubgroup = 8192;
```

Comment:
Why was this comment removed? I guess it applied to all supported GFX targets?

Reply:
Yes, when this comment was written, we didn't support GFX9+ on LLPC. Thus, it is outdated now.